opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -45,6 +51,101 @@
 
 _logger = logging.getLogger(__name__)
 
+_DEFAULT_OTEL_ATTRIBUTE_COUNT_LIMIT = 128
+_ENV_VALUE_UNSET = ""
+
+
+class LogLimits:
+    """This class is based on a SpanLimits class in the Tracing module.
```

Comment:
```suggestion
    f"""This class is based on a SpanLimits class in the Tracing module.
```

This is necessary for `{_DEFAULT_ATTRIBUTE_COUNT_LIMIT}` to be rendered properly.

Reply:
If I make this change, I get an error from the linter `Statement seems to have no effect Pylint(W0104:pointless-statement)`. In the referenced class `SpanLimits` the description is not formatted despite listing variables. Should I explicitly disable the pylint check in this case?

Reply:
Resolved this conversation, no point in delaying this PR for this isse.